OutOfSchool/OutOfSchool.AuthServer.Tests/EmailSender/SendGridAccessibilityServiceTests.cs

```diff
@@ -83,4 +83,32 @@ public void IsSendGridAccessible_WithOptionsTimeoutTimeNull_ReturnsTrueAfterTime
         Assert.True(earlyResult);
         Assert.True(result);
     }
+
+    [Test]
+    public void GetAccessibilityTime_WithAccessibleSendGrid_ReturnsDateTimeOffsetNow()
+    {
+        // Arrange
+        var now = DateTimeOffset.Now;
+
+        // Act
+        var accessTime = _emailService.GetAccessibilityTime(now);
+
+        // Assert
+        Assert.AreEqual(now, accessTime);
+    }
+
+    [Test]
+    public void GetAccessibilityTime_WithInaccessibleSendGrid_ReturnsAccessibilityTime()
+    {
+        // Arrange
+        var now = DateTimeOffset.Now;
+
+        // Act
+        _emailService.SetSendGridInaccessible(now);
```

Comment:
(Small naming nuance) Technically saying `.SetSendGridInaccessible()` is a related to an `Arrange` part, not `Act`.